src/EntryList.js

```diff
@@ -0,0 +1,13 @@
+function EntryList() {
+
+    return (
+        <div>
+            <label>My List</label>
+            <tr>
+                <th>Enter Something</th>
```

Comment:
Please don't add code that you don't know what's is doing

you added tr tag which belongs to table tag

Reply:
I wanted to add a cod for rows